workflow/rocoto/workflow_tasks.py

```diff
@@ -326,8 +322,10 @@ def aerosol_init(self):
 
         dependencies = rocoto.create_dependency(dep_condition='and', dep=deps)
 
+        cycledef = 'gfs_cont'
```

Comment:
suggest `ctrl` if this is meaning 'gfs control

Reply:
Changed to `gfs_seq`.